src/frontend/src/components/ProjectDetailsV2/TaskSectionPopup.tsx

```diff
@@ -74,7 +78,9 @@ const TaskSectionPopup = ({ taskId, body, feature }: TaskSectionPopupPropType) =
         <AssetModules.DescriptionOutlinedIcon
           style={{ width: '20px' }}
           className="hover:fmtm-text-primaryRed"
-          onClick={() => {}}
+          onClick={() => {
+            navigate(`/project-submissions/MTAwMTExMTE=?tab=table&task_id=${taskId}`);
```

Comment:
Is this navigating to a hardcoded project url?
Shouldn't it be dynamic based on the current project page, or am I missing something?